### DISCUSSION TITLE
Why is this not a valid GUID?

### DISCUSSION THREAD

**Discussion by matthiaslischka** [uuidjs/uuid#674] [Q&A] [answered]
### Before you begin...

- [X] I have searched the existing issues
- [X] I am not using version 13.x of node  (if so, please upgrade)

### Description of the problem

"49C1577D-2CB7-41BB-1CF3-08D960A71AAF" should be a valid GUID imho.

### Recipe for reproducing

```bash
import { validate } from "uuid";

const isValid1 = validate("49C1577D-2CB7-41BB-1CF3-08D960A71AAF"); // <---- ?!
const isValid2 = validate("93595E16-CE95-4ACD-80F4-66CE17C6D906");
console.log({ isValid1 }); // false
console.log({ isValid2 }); // true
```

### Additional information

https://codesandbox.io/s/typescript-playground-export-forked-lecdtm

I checked multiple online validators and all said that this Guid is valid. Dunno what's going on.

THX and BR,
Matthias

**Comment by broofa** [comment, marked as answer]
Invalid `variant` field.
 
The "1" in `1CF3` must be 8, 9, A, or B, per https://www.rfc-editor.org/rfc/rfc4122#section-4.1.1

**Reply by matthiaslischka** [reply]
Yes, sry, I was just figuring this out after I created the ticket and wanted to close this issue already. It's strange how not a single validator I tested had a problem with that and I have to check where we got that Id from. 😅

Thanks a lot.

**Reply by Borewit** [reply]
That GUID is a **valid** Windows GUID, which is **not** the same as a RFC-4122 UUID.
